share/rocm/cmake/ROCMSetupVersion.cmake

```diff
@@ -132,12 +148,31 @@ function(rocm_setup_version)
         if(PARSE_VERSION MATCHES "^[0-9]+\\.[0-9]+$")
             set(PARSE_VERSION ${PARSE_VERSION}.0)
         endif()
-        if(PARSE_NO_GIT_TAG_VERSION)
-            set(PACKAGE_VERSION ${PARSE_VERSION})
-        else()
+        if(PARSE_ADD_GIT_TAG_VERSION)
             rocm_get_commit_count(COMMIT_COUNT PARENT ${PARSE_PARENT})
             rocm_get_version(PACKAGE_VERSION VERSION ${PARSE_VERSION}.${COMMIT_COUNT})
+        else()
+            set(PACKAGE_VERSION ${PARSE_VERSION})
         endif()
+
+        rocm_get_patch_version(ROCM_VERSION_NUM)
+        if("${ROCM_VERSION_NUM}" STREQUAL "")
+            set(ROCM_VERSION_NUM "99999")
+        endif()
+        set(PACKAGE_VERSION ${PACKAGE_VERSION}.${ROCM_VERSION_NUM})
```

Comment:
The rocm version should be added based on the build id.

Reply:
This is not required.